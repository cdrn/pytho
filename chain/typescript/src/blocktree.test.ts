import { assert } from 'console'
import BlockTree from './blocktree'

describe('Test the blocktree functionality', () => {
  test('the tests run', () => {
    expect(true)
  })
  test('Instantiating a blockchain makes a genesis block', () => {
    const blocktree = new BlockTree
    console.log(blocktree)
  })
  test('', () => {

  })
})