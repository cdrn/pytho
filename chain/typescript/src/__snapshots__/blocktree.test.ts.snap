// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the blocktree functionality Instantiating a blockchain makes a genesis block 1`] = `
BlockTree {
  "root": Object {
    "depth": 0,
    "parent": null,
    "post": Object {
      "content": "the ur post",
    },
    "previousHash": "1",
    "proof": "100",
    "timestamp": 1611168618640,
  },
}
`;
